packages/utility/src/is-defined.ts

```diff
@@ -0,0 +1,3 @@
+export function isDefined<T extends unknown>(value: T | undefined | null): value is T {
```

Comment:
Please refactor this to a subfolder `typeguards`

Reply:
cool

Reply:
fixed